exp/containers.md

```diff
@@ -26,28 +26,36 @@ All containers in Anbox Cloud are ephemeral, which means that as soon as a conta
 
 ## Container life cycle
 
-When you create a container by launching an application or an image, it executes the following steps in order:
+When you [create a container](https://discourse.ubuntu.com/t/launch-a-container/24327) by either launching or initialising an application or an image, it executes the following steps in order:
 
+1. Schedule a container on LXD node.
```

Comment:
The "it" in "it executes the following steps in order" refers to the container - that doesn't fit with the first step here, since the container probably doesn't schedule itself. ;)
So either change the "it" to "LXD" and make it clearer in the following steps which happen inside the container and which outside, or move the first step into the introduction. (For example: "... by either launching or initialising an application or an image, LXD schedules the container on a LXD node. The container then executes ..."

Reply:
Changing the "it" to "LXD" doesn't sound right to me considering the context. E.g. Execute the hook, it's performed by the container itself rather than the LXD. I'd vote to move the first step into the introduction as that sounds more reasonable and natural